codecept.conf.js

```diff
@@ -0,0 +1,90 @@
+const {
+    SAUCE_USERNAME,
+    SAUCE_ACCESS_KEY,
+    TRAVIS_JOB_NUMBER,
+    CI,
+    BROWSER_NAME = 'chrome',
+    BROWSER_VERSION = 'latest',
+    BROWSER_PLATFORM,
+    PLATFORM_VERSION,
+    DEVICE_NAME,
+    DEFAULT_WAIT_TIME = 9000
+} = process.env;
+const MOBILE_PLATFORMS = ['iOS', 'Android'];
+
+// Local selenium config
+const commonConfigObj = {
+    browser: BROWSER_NAME,
+    url: 'http://localhost:8080',
+    smartWait: DEFAULT_WAIT_TIME,
+    restart: true,
+    waitForTimeout: DEFAULT_WAIT_TIME
+};
+
+const helperObj = {};
+const isLocalBuild = typeof SAUCE_USERNAME === 'undefined';
+
+if (isLocalBuild) {
+    // // Config to test using the Chrome emulator
+    // commonConfigObj.desiredCapabilities = {
+    //     platform: 'iOS',
+    //     platformName: BROWSER_PLATFORM,
+    //     browserName: 'chrome',
+    //     platformVersion: '11.2',
+    //     deviceName: 'iOS',
+    //     deviceOrientation: 'portrait',
+    //     chromeOptions: {
+    //         mobileEmulation: {
+    //             deviceName: 'iPhone 6/7/8'
+    //         }
+    //     }
+    // };
+    helperObj.WebDriverIO = commonConfigObj;
+} else {
+    // Common saucelab config
+    const sauceObj = {
```

Comment:
Can this be moved to a higher level variable, or does it need to be created on each run of this code?